src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -989,6 +989,42 @@ let ``Uses documentation`` () =
     given ["-h"] |> yields message
     given ["-?"] |> yields message
 
+[<Fact>]
+let ``Shows help text for generateazurepayload command`` () =
+    let name = Path.GetFileNameWithoutExtension (Assembly.GetEntryAssembly().Location)
+    let message =
+        name
+        |> sprintf "Usage:
+                      %s generateazurepayload [options]
+
+                    Options:
+                      -n <n> (REQUIRED)                                   A number.
```

Comment:
The fragment `A number` is not very informative. Some more info wouldn't harm about the purpose, range of values, type (floating point?)...

Reply:
Updated comment description to be `An integer`.